scripts/pre_commit_linter.py

```diff
@@ -2416,25 +2685,69 @@ def _lint_all_files(self):
         summary_messages = []
 
         result_queues = [
-            css_in_html_result, css_result, py_result]
+            css_in_html_result, css_result, py_result,
+            py_result_for_python3_compatibility]
 
         for result_queue in result_queues:
             while not result_queue.empty():
                 summary_messages.append(result_queue.get())
 
         with _redirect_stdout(_TARGET_STDOUT):
-            print '\n'.join(summary_messages)
-            print ''
+            python_utils.PRINT('\n'.join(summary_messages))
+            python_utils.PRINT('')
 
         return summary_messages
 
+    def _check_division_operator(self):
```

Comment:
Should this include checks for e.g. the print, map, etc. operators? Perhaps better to do it this way rather than use the string representation, assuming you're already calculating the AST.

Reply:
I thought about this but they will all be of type 'ast.Call' objects so it cannot distinguish between print and python_utils.print. For this, the regex expression works fine.